ansible/roles/test/tasks/sonic.yml

```diff
@@ -62,8 +62,12 @@
   include: arpall.yml
   tags: arp
 
-### When calling this basic_route, Chip is setup with proper parameters and
-### SAI API used by syncd daemon are called. This test is mainly for validation purpose used by external vendors.
+- name: Test hardware configuration
```

Comment:
suggest to have a better name: test port alias configuration, and also rename the playbook name to port-alias.yml